test/integration/testdata/golden.v2.json

```diff
@@ -7,6 +7,9 @@
   "paths": {
    "/test/custom/bac": {
     "get": {
+     "produces": [
+      "application/json"
```

Comment:
The old test was incomplete and swagger recommends having this produces field the response includes a schema. Since we are sharing the input structs for v2 and v3, I added this field for v2 tests as well. See [link](https://swagger.io/specification/v2/#responseObject)

"A definition of the response structure. It can be a primitive, an array or an object. If this field does not exist, it means no content is returned as part of the response. As an extension to the Schema Object, its root type value may also be "file". This SHOULD be accompanied by a relevant produces mime-type."